docs/UserGuide.adoc

```diff
@@ -123,7 +123,7 @@ a person and the criteria for valid inputs:
 
 * tt/**TISSUE TYPE** - the type of tissues
 ** Applicable to: patients and donors
-** Valid input: a string of 6 numbers (from 1 - 12), separated by commas
+** Valid input: a string of 6 unique numbers (from 1 - 12), separated by commas
```

Comment:
Can say order does not matter too? Will be clearer. Thanks

Reply:
Added.